md-rest-api/src/main/java/com/wipro/ats/bdre/md/rest/ProcessAPI.java

```diff
@@ -890,9 +890,9 @@ public RestWrapper importData(@ModelAttribute("fileString")
     }
 
     @RequestMapping(value = {"/execute", "/execute/"}, method = RequestMethod.POST)
-    @ResponseBody
-    public RestWrapper executeProcess(@ModelAttribute("process")
-                                      @Valid Process process, Principal principal) {
+    @ResponseBody public
+    RestWrapper executeProcess(@ModelAttribute("process")
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove the unused method parameter(s) "bindingResult". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS1172)